scipy/_lib/_uarray/__init__.py

```diff
@@ -33,7 +33,7 @@
 creating multimethods are mostly orthogonal activities, and knowing
 one doesn't necessarily require knowledge of the other, although it
 is certainly helpful. We expect core API designers/specifiers to write the
-multimethods, and implementors to override them. But, as is often the case,
+multimethods, and implementers to override them. But, as is often the case,
```

Comment:
Same here.

Reply:
The word `implementor` is missing from some dictionaries (including the OED but it lacks `implementer` too):
* [OED](https://www.oed.com/search/dictionary/?q=implementor)
* [Oxford Learner's](https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=implementor)
* [Cambridge](https://dictionary.cambridge.org/fr/spellcheck/anglais/?q=implementor)
* [Longman](https://www.ldoceonline.com/spellcheck/english/?q=implementor)

However, I can find it as a variant of `implementer` in these dictionaries:
* [Collins](https://www.collinsdictionary.com/spellcheck/english?q=implementor)
* [Merriam-Webster](https://www.merriam-webster.com/dictionary/implementor)

The Google n-gram viewer shows that implementer is by far more common in contemporary English:
https://books.google.com/ngrams/graph?content=implementor%2Cimplementer&year_start=1945&year_end=2019&corpus=en-2019

For the above reason, it is missing from the SCWOL word lists used by _aspell_, which I see as the open-source reference for spelling:
http://app.aspell.net/lookup?dict=en_US-large;words=implementor

The rationale for SCOWL is to avoid rare variants, for readability.